opentext-page.html

```diff
@@ -0,0 +1,482 @@
+                        <li><a href="#Developer">Developer Cloud</a></li>
+                    </ul>
+                </div>
+            </div>        
+        </div>
+        <div class="container-3">
+            <div class="row">
+                <div class="column">
+                    <ul>
+                        <li><a href="#Business">Business Network Cloud</a></li>
+                        <li><a href="#Security">Security & Protection Cloud</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>    
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Accelerate">Accelerate customer communication</a>                                
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Reduce">Reduce eDiscovery costs</a>                                                
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Protect">Protect patient data</a>                
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Achieve">Achieve website management ROI</a>               
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#L'oreal">L'oreal logo</a>
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Sprint">Sprint logo</a>
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Philips">Philips logo</a>                
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Vodafone">Vodafone logo</a>                               
+                </div>
+            </div>
+            <button type="submit">Read more Sucess stories</button>
+        </div>
+        <div class="blog">
+            <div class="row">
+                <div class="column">
+                    <a href="#News">Latest news and insights</a>
+                        <!-- no borders with an arrow -->
+                        <button type="submit">See All</button>
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="blog">
```

Comment:
same issue... you keep dropping the use of container class
fix all instances.... stopping now from pointing all of them out...

Reply:
What do you mean by container class?

Reply:
Line 86 is an example.
You keep dropping these and instead either doing something like line 327, or an HTML element tag like `<footer>`.
In reality, you only ever truly have one container class, but for the sake of consistency, I am asking you to add them here.  In your next page, we will go over the structure for container > row > column again.